Formula/usbredir.rb

```diff
@@ -0,0 +1,37 @@
+class Usbredir < Formula
+  desc "USB traffic redirection library"
+  homepage "https://www.spice-space.org"
+  url "https://www.spice-space.org/download/usbredir/usbredir-0.8.0.tar.bz2"
+  sha256 "87bc9c5a81c982517a1bec70dc8d22e15ae197847643d58f20c0ced3c38c5e00"
+
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "libusb"
+
+  def install
+    # https://stackoverflow.com/questions/15860127/how-to-configure-tcp-keepalive-under-mac-os-x
+    inreplace "usbredirserver/usbredirserver.c" do |s|
+      s.gsub! "SOL_TCP", "IPPROTO_TCP"
+      s.gsub! "TCP_KEEPIDLE", "TCP_KEEPALIVE"
```

Comment:
This needs to be reported and fixed upstream. We ship software that builds natively on macOS.

Reply:
Reported [upstream](https://gitlab.freedesktop.org/spice/usbredir/issues/9).

Reply:
Patch [submitted](https://lists.freedesktop.org/archives/spice-devel/2019-January/047521.html) upstream.

Reply:
Patch merged upstream, formula updated to apply upstream patch ahead of next release.